apps/researcher/src/app/[locale]/objects/[id]/(provenance)/actions.ts

```diff
@@ -56,7 +67,27 @@ export async function addProvenanceEnrichment({
   location,
   community,
   qualifier,
+  motivations,
 }: AddProvenanceEnrichmentProps) {
+  const tEnQualifier = await getTranslations({
+    locale: 'en',
+    namespace: 'QualifierSelector',
+  });
+
+  const tEnType = await getTranslations({
+    locale: 'en',
+    namespace: 'ProvenanceEventType',
+  });
+  const motivationsWithValues = Object.entries(motivations).reduce(
+    (list, [key, motivation]) => {
```

Comment:
Minor: the name `list` is very generic - it could be any list. To make its meaning clearer, use a functional name instead?